.travis.yml

```diff
@@ -14,12 +14,14 @@ addons:
 
 php:
   - 7.2
+  - 7.3
   - 7.4
+  - 8.0
+  - 8.1
 
 matrix:
   allow_failures:
-    - php: 7.0
-    - php: 7.4
+    - php: 8.1
```

Comment:
We had this configured so our production version would not be allowed to fail (PHP 7.2) But having it configured like this is fine for now, I see no reason why any php 7 version would start failing in the near feature.

On a side note. If we move away all code quality tests from Travis to Github Actions, we would be able to shut down Travis. Also see my previous comment for some suggestions.

Reply:
I removed the 3 files beloging to travis